Appveyor build failing

The appveyor build is failing fairly consistently on the python `3.5` build (eg. see https://ci.appveyor.com/project/matplotlib/matplotlib/build/1.0.2957/job/dqhano9lykge7ffb#L1375). The actual failure is somewhere in `test_artist`:

```
lib\matplotlib\tests\test_artist.py ...ssCommand exited with code -1073740791
```

Probably related: I see many `Windows fatal exception: code 0xe06d7363` in `test_artist` on Python 3.6.3 but not on Python 3.6.1. The tracebacks seem to point to the `_path` module. Exception Code `0xE06D7363` hints at [uncaught C++ exception](https://blogs.msdn.microsoft.com/oldnewthing/20100730-00/?p=13273/). So far I have not been able to reproduce these exceptions outside the tests or in a debugger. Maybe downgrading the Python 3.5 and 3.6 versions in Appveyor will work around this issue.

```
>C:\python-3.6.3-amd64\Scripts\py.test.exe C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py -v --capture=no
C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\__init__.py:1373: UserWarning: Matplotlib is not built with the correct FreeType version to run tests.  Set local_freetype=True in setup.cfg and rebuild. Expect many image comparison failures below. Expected freetype version 2.6.1. Found freetype version 2.8.1. Freetype build type is not local
  "" if ft2font.__freetype_build_type__ == 'local' else "not "
============================= test session starts =============================
platform win32 -- Python 3.6.3, pytest-3.2.2, py-1.4.34, pluggy-0.4.0 -- c:\python-3.6.3-amd64\python.exe
cachedir: ..\..\..\.cache
rootdir: C:\, inifile:
collected 13 items

..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_patch_transform_of_none PASSED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_collection_transform_of_none PASSED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_clipping[png] PASSED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_clipping[pdf] Windows fatal exception: code 0xe06d7363

Current thread 0x00002014 (most recent call first):
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1511 in pathOperations
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1521 in writePath
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1719 in draw_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\patches.py", line 574 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\spines.py", line 317 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\axes\_base.py", line 2399 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1295 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 2592 in print_pdf
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backend_bases.py", line 2259 in print_figure
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1814 in savefig
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 288 in compare
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 401 in wrapper
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 143 in pytest_pyfunc_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 1171 in runtest
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 112 in pytest_runtest_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 196 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in call_runtest_hook
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 162 in call_and_report
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 82 in runtestprotocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 68 in pytest_runtest_protocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 169 in pytest_runtestloop
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 146 in _main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 110 in wrap_session
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 139 in pytest_cmdline_main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\config.py", line 58 in main
  File "C:\python-3.6.3-amd64\Scripts\py.test.exe\__main__.py", line 9 in <module>
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 85 in _run_code
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 193 in _run_module_as_main
Windows fatal exception: code 0xe06d7363

Current thread 0x00002014 (most recent call first):
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1511 in pathOperations
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1521 in writePath
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 1719 in draw_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\patches.py", line 574 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\spines.py", line 317 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\axes\_base.py", line 2399 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1295 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_pdf.py", line 2592 in print_pdf
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backend_bases.py", line 2259 in print_figure
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1814 in savefig
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 288 in compare
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 401 in wrapper
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 143 in pytest_pyfunc_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 1171 in runtest
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 112 in pytest_runtest_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 196 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in call_runtest_hook
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 162 in call_and_report
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 82 in runtestprotocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 68 in pytest_runtest_protocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 169 in pytest_runtestloop
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 146 in _main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 110 in wrap_session
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 139 in pytest_cmdline_main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\config.py", line 58 in main
  File "C:\python-3.6.3-amd64\Scripts\py.test.exe\__main__.py", line 9 in <module>
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 85 in _run_code
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 193 in _run_module_as_main
Windows fatal exception: code 0xe06d7363

<snip thousands of lines>

Current thread 0x00002014 (most recent call first):
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 562 in _convert_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 570 in draw_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\patches.py", line 574 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\spines.py", line 317 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\axes\_base.py", line 2399 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1295 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 1248 in _print_svg
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 1193 in print_svg
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backend_bases.py", line 2259 in print_figure
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1814 in savefig
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 288 in compare
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 401 in wrapper
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 143 in pytest_pyfunc_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 1171 in runtest
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 112 in pytest_runtest_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 196 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in call_runtest_hook
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 162 in call_and_report
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 82 in runtestprotocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 68 in pytest_runtest_protocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 169 in pytest_runtestloop
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 146 in _main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 110 in wrap_session
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 139 in pytest_cmdline_main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\config.py", line 58 in main
  File "C:\python-3.6.3-amd64\Scripts\py.test.exe\__main__.py", line 9 in <module>
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 85 in _run_code
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 193 in _run_module_as_main
Windows fatal exception: code 0xe06d7363

Current thread 0x00002014 (most recent call first):
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 562 in _convert_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 570 in draw_path
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\patches.py", line 574 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\spines.py", line 317 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\axes\_base.py", line 2399 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\image.py", line 138 in _draw_list_compositing_images
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1295 in draw
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\artist.py", line 55 in draw_wrapper
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 1248 in _print_svg
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backends\backend_svg.py", line 1193 in print_svg
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\backend_bases.py", line 2259 in print_figure
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\figure.py", line 1814 in savefig
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 288 in compare
  File "C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py", line 401 in wrapper
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 143 in pytest_pyfunc_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\python.py", line 1171 in runtest
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 112 in pytest_runtest_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 196 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 182 in call_runtest_hook
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 162 in call_and_report
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 82 in runtestprotocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\runner.py", line 68 in pytest_runtest_protocol
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 265 in __init__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 248 in _wrapped_call
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 613 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 169 in pytest_runtestloop
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 146 in _main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 110 in wrap_session
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\main.py", line 139 in pytest_cmdline_main
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 614 in execute
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 334 in <lambda>
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 339 in _hookexec
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\vendored_packages\pluggy.py", line 745 in __call__
  File "c:\python-3.6.3-amd64\lib\site-packages\_pytest\config.py", line 58 in main
  File "C:\python-3.6.3-amd64\Scripts\py.test.exe\__main__.py", line 9 in <module>
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 85 in _run_code
  File "c:\python-3.6.3-amd64\lib\runpy.py", line 193 in _run_module_as_main
SKIPPED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_remove PASSED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_default_edges[png] FAILED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_properties PASSED
..\..\..\python-3.6.3-amd64\Lib\site-packages\matplotlib\tests\test_artist.py::test_setp PASSED

================================== FAILURES ===================================
___________________________ test_default_edges[png] ___________________________

expected = 'C:\\Users\\gohlke\\test\\result_images\\test_artist\\default_edges-expected.png'
actual = 'C:\\Users\\gohlke\\test\\result_images\\test_artist\\default_edges.png'
tol = 0

    def _raise_on_image_difference(expected, actual, tol):
        __tracebackhide__ = True

        err = compare_images(expected, actual, tol, in_decorator=True)

        if not os.path.exists(expected):
            raise ImageComparisonFailure('image does not exist: %s' % expected)

        if err:
            for key in ["actual", "expected"]:
                err[key] = os.path.relpath(err[key])
            raise ImageComparisonFailure(
                'images not close (RMS %(rms).3f):\n\t%(actual)s\n\t%(expected)s '
>                % err)
E           matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 3.039):
E               result_images\test_artist\default_edges.png
E               result_images\test_artist\default_edges-expected.png

C:\python-3.6.3-amd64\Lib\site-packages\matplotlib\testing\decorators.py:211: ImageComparisonFailure
=============== 1 failed, 8 passed, 4 skipped in 16.30 seconds ================
```

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -95,6 +95,12 @@ install:
   # pytest-cov>=2.3.1 due to https://github.com/pytest-dev/pytest-cov/issues/124
   - pip install -q pytest "pytest-cov>=2.3.1" pytest-rerunfailures pytest-timeout
 
+  # Apply patch to `subprocess` on Python versions > 2 and < 3.6.3
+  # https://github.com/matplotlib/matplotlib/issues/9176
+  - python -c "import sys; sys.exit(not (3,) < sys.version_info < (3,6,3))" && (
+    curl -sL https://github.com/python/cpython/pull/1224.patch |
+    patch -fsup 1 -d %CONDA_PREFIX% ) || ( set errorlevel= )
+
   # Let the install prefer the static builds of the libs
   - set LIBRARY_LIB=%CONDA_PREFIX%\Library\lib
   - mkdir lib || cmd /c "exit /b 0"

